test/test-framework/cmd/manager/main.go

```diff
@@ -0,0 +1,65 @@
+package main
```

Comment:
Not sure, if it makes sense to add this in test-framework. I'd rather have this in test/e2e directory so that we are clear that this is part of test suite not framework as such or else it may confuse new developers contributing to the repo.

Reply:
The way that the repo is designed, users shouldn't be using anything outside of `pkg` for use as a library. The `test` directory is where we keep all of the SDK's own tests and we further divided it based on what we are testing. `e2e` contains our e2e tests and uses its own framework (which is similar to the exposed framework in `pkg/test` but slightly modified). We have the `test-framework` directory containing our tests to make sure the test-framework works properly, just like we have the `ansible-memcached` directory there to test the ansible operator.

We make it pretty clear in the documentation that the test framework is in `pkg/test`, so I believe the layout that we currently have with `test/test-framework` is reasonable (and we've had this directory since the test framework was introduced and it doesn't seem like that has caused any problems so far).

Reply:
Alright, thank you @AlexNPavel